CONTRIBUTING

```diff
@@ -0,0 +1,102 @@
+# How to contribute
+
+Welcome to `digits` library for GoLang.
+
+###### TODO <@joshuanario, insert blurb here about ECKSBEE and the library.>

```

Comment:
I feel like this file should not mention EDGARBrowser or ecksbee. Or at the very least, it should briefly mention `telefacts` as currently the biggest adopter of `digits`, which can be seen in production at edgarbrowser.com.

Under resources, XBRL technical specification for `precision` and `decimals` is a must as it is definitely the main inspiration for creating this library.  https://www.xbrl.org/WGN/precision-decimals-units/WGN-2017-01-11/precision-decimals-units-WGN-2017-01-11.html#rounding-table

Mentioning IEEE-754 (wiki) under resources should also be done as achieving octuple precision is a goal for `digits`.  https://en.wikipedia.org/wiki/IEEE_754

Honestly, those two links are what I would put under the resources section for this file.